contrib/pkg/createcluster/create.go

```diff
@@ -118,13 +118,18 @@ const (
 // NewCreateClusterCommand creates a command that generates and applies cluster deployment artifacts.
 func NewCreateClusterCommand() *cobra.Command {
 	var homeDir = "."
+	var defaultPullSecretFile string
+
 	if u, err := user.Current(); err == nil {
 		homeDir = u.HomeDir
 	}
 	defaultSSHPublicKeyFile := filepath.Join(homeDir, ".ssh", "id_rsa.pub")
-	defaultPullSecretFile := filepath.Join(homeDir, ".pull-secret")
 	defaultAWSCredsFile := filepath.Join(homeDir, ".aws", "credentials")
 
+	if _, err := os.Stat(filepath.Join(homeDir, ".pull-secret")); err == nil {
```

Comment:
My preference would be to do it like the following so that we are not duplicating the code of building out the path.
```
defaultPullSecretFile := filepath.Join(homeDir, ".pull-secret")
if _, err := os.Stat(defaultPullSecretFile); err != nil {
  defaultPullSecretFile = ""
}
```